### DISCUSSION TITLE
[help]

### DISCUSSION THREAD

**Discussion by modullll1** [Chia-Network/chia-blockchain#3171] [General]
У меня вообще не подключается к кошельку. Крутится постоянно подключение к кошельку и все. Что делать?
Пробовал переустановить программу не помогло. Перезагружал компьютер не помогло. Раньше все работало
![Диспетчер задач](https://user-images.githubusercontent.com/83408910/116555791-7226ad00-a905-11eb-9c53-dfb856932292.jpg)

**Comment by maomingyang1314** [comment]
You may have used HDD for drawing, causing memory overflow. You should use SSD for drawing
